pontoon/base/templates/translate.html

```diff
@@ -125,9 +125,10 @@
             {% endif %}
 
             {% if user.is_authenticated() %}
+            <li><a href="{{ url('socialaccount_connections') }}">Connect your account</a></li>
```

Comment:
This needs to be visible only if you haven't merged FxA and Persona accounts yet.